packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -38,17 +38,17 @@ const propTypes = {
     GranularityOptions.DATE,
```

Comment:
If we are doing a major version bump, I vote to change the exported enums to strings. 
### Before
``` jsx
<OnsetPicker granularity={OnsetPicker.Opts.Granularities.AGE} precisionSet={OnsetPicker.Opts.Precisions.ONAT} />
```
### After
``` jsx
<OnsetPicker granularity="age" precisionSet="on/at" />
```
We had discussion about this here: https://github.com/cerner/terra-core/issues/1044

Reply:
So pretty much just remove the exported enums and update docs? Easy enough.